plugin/src/main/java/org/opensearch/ml/utils/error/ErrorMessage.java

```diff
@@ -44,14 +44,18 @@ public ErrorMessage(Throwable exception, int status) {
     }
 
     private String fetchType() {
-        return exception.getClass().getSimpleName();
+        return exception == null ? "NULL" : exception.getClass().getSimpleName();
```

Comment:
If exception is null, should we return empty exception type ?

Reply:
what about `No Exception Provided` or `Unknown Exception`?

Reply:
Sounds good, I will use `Unknown Exception`.